corehq/apps/domain/tests/test_deletion_models.py

```diff
@@ -57,6 +57,7 @@
     'util.PermanentBounceMeta',
     'util.TransientBounceEmail',
     'registration.AsyncSignupRequest',
+    'users.UserHistory',
```

Comment:
If a domain is deleted, we would not want to lose the audit of users? That information would still be useful right, as an "audit"

Reply:
I'm not sure what expectations there are for being able to audit a deleted domain, I think that's a question for product. That also reminds me, you might need to add deletion of this data to the [GDPR scrub process](https://confluence.dimagi.com/display/GTD/How+to+scrub+a+user+for+GDPR+compliance), although product should confirm. I'm not sure how GDPR requirements for deletion intersect with HIPAA requirements for retention.

Reply:
https://docs.google.com/document/d/1Z5ZOu7O-BVOh62oCTc1JjNRsBAiMzs9i8Cf48q_9BDQ/edit?disco=AAAAMinwpI4

Reply:
Cool, thank you - just confirming that you're also checking on GDPR specifically?

Reply:
Added that as well here, https://docs.google.com/document/d/1Z5ZOu7O-BVOh62oCTc1JjNRsBAiMzs9i8Cf48q_9BDQ/edit?disco=AAAAM1WMoiI

Reply:
@orangejenny thank you for bringing both of them to attention

Reply:
Confirmed with Clayton that its okay for us to retain the logs, even if the domain is deleted.

Quoting him,
> It would probably be safest if we had the option to remove them or not and also to independently remove them later.

So going to leave this as it is. And update the confluence/document needed to reflect any changes.
following up on email thread were this is being discussed, also adding Jenny and Simon as FYIs.

Reply:
Thank you for the thorough followup on this.